package.json

```diff
@@ -107,7 +107,8 @@
     "vitepress": "catalog:",
     "vitest": "catalog:",
     "vitest-browser-vue": "^0.0.1",
-    "vue": "catalog:"
+    "vue": "catalog:",
+    "vue-tsc": "^2.2.0"
```

Comment:
still not sure how to add to catalog 😄

Reply:
add it to `catalog:` entry in `pnpm-workspace.yaml` file

Reply:
can you also update `vitest-browser-vue`?

Reply:
https://pnpm.io/catalogs

Reply:
I thought there is a command to add it. I will add it manually

Reply:
https://github.com/pnpm/pnpm/issues/8876

Reply:
> can you also update `vitest-browser-vue`?

https://www.npmjs.com/package/vitest-browser-vue?activeTab=versions

`vitest-browser-vue@0.0.1` is latest

Reply:
I mean, move the version to pnpm file